updater/lib/dependabot/updater/operations/create_security_update_pull_request.rb

```diff
@@ -12,6 +12,17 @@ module Operations
       class CreateSecurityUpdatePullRequest
         include SecurityUpdateHelpers
 
+        class SecurityUpdateError < StandardError
```

Comment:
should we call this `TransitiveSecurityUpdateError` for specificity?

Reply:
i matched with class name convention, specificity would be a good idea for sure. ill wait for someone to review once to be sure in case im missing anything.